src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -1564,6 +1564,14 @@ public sealed class ValidateSetAttribute : ValidateEnumeratedArgumentsAttribute
         private static ConcurrentDictionary<Type, IValidateSetValuesGenerator> s_ValidValuesGeneratorCache = new ConcurrentDictionary<Type, IValidateSetValuesGenerator>();
 
         /// <summary>
+        /// If AllowAll==true 'ValidateElement' always pass all elements.
+        /// Useful if we need enable IntelliSense (TabComplete) but effectively disable validation
+        /// when a parameter can accept not only the listed values but also arbitrary.
+        /// Ex.: Get-Date -Format
+        /// </summary>
+        public bool AllowAll { get; set; } = false;
```

Comment:
This doesn't feel right to me - you shouldn't use a `Validation` attribute if you aren't validating.

Reply:
This should be reverted.